.github/workflows/nightly-windows.yml

```diff
@@ -77,11 +77,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
```

Comment:
any chance to upgrade this as well?

Reply:
Lets address separately - https://github.com/hazelcast/hazelcast-cpp-client/issues/1232